geomesa-convert/geomesa-convert-parquet/src/main/scala/org/locationtech/geomesa/convert/parquet/ParquetFunctionFactory.scala

```diff
@@ -58,6 +60,8 @@ class ParquetFunctionFactory extends TransformerFunctionFactory {
     }
 
     protected def eval(x: U, y: U): T
+
+    protected def eval(b: Array[Byte]): T = WKBUtils.read(b).asInstanceOf[T]
```

Comment:
we can probably leave these functions alone, and just note them as deprecated in the user docs - we can get equivalent functionality with a combo of avroPath + our regular geometry parsing functions (from geomesa-convert-common)